File: Web_Programming_Technologies/React/3feb/Calculator.js
import React from "react";
export default class Calculator extends React.Component {

    render() {
        const num1 = parseInt(this.props.num1);
        const num2 = parseInt(this.props.num2);
        const op = this.props.op;
        let result = 0;
        if (op === "+") {
            result = num1 + num2;
        }
        if (op === "-") {
            result = num1 - num2;
        }
        if (op === "/") {
            result = num1 / num2;
        }
        if (op === "*") {
            result = num1 * num2;
        }
        return <div><p>{num1} {op} {num2} = {result}</p></div>
    }


}

File: MS.Net/23feb/Ecommerce/ExtensionDemoApp/Calculator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ExtensionDemoApp
{
    public sealed class Calculator
    {
        public int Result { get; set; }

        public void Addition(int num1, int num2)
        {
            Result = num1 + num2;
        }

        public void Subtraction(int num1, int num2)
        {
            Result = num1 - num2;
        }
    }
}